update arista_eos_show_inventory

yshu95 opened the pull request:
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT
arista_eos_show_inventory

##### SUMMARY
I couldn't parse of arista_eos_show_inventory.
I think that, in netmiko at arista_eos, output of "show inventory" has space on end of line.
e.g.) netmiko send_command result
```
 'System has 4 fan modules\n'
 '  Module  Number of Fans  Model            Serial Number    \n'
 '  ------- --------------- ---------------- ---------------- \n'
 '  1       1               FAN-7000-R       N/A              \n'
 '  2       1               FAN-7000-R       N/A              \n'
 '  3       1               FAN-7000-R       N/A              \n'
 '  4       1               FAN-7000-R       N/A              \n'
```
So I updated arista_eos_show_inventory.textfsm.

And i think that recent "show inventory" command output has "Epoch" section.
e.g.)
```
 '  HW Version  Serial Number  Mfg Date   Epoch \n'
 '  ----------- -------------- ---------- ----- \n'
 '  11.12       XXXXXXXXXX    2017-04-21 01.00 \n'
```
So I updated arista_eos_show_inventory.raw.

test
```
tests/test_structured_data_against_parsed_reference_files.py::test_raw_data_against_mock[tests/arista_eos/show_inventory/arista_eos_show_inventory.raw] PASSED [  2%]
```

I'm sorry in my poor English.

FragmentedPacket commented on templates/arista_eos_show_inventory.textfsm:
```diff
@@ -11,26 +11,26 @@ Chassis
 Power_Supply
   ^\s+Slot
   ^\s+-
-  ^\s+${PORT}\s+${NAME}\s+${SN}$$ -> Record
```
What if you just got rid of the end of line requirement?

e.g. removing the `$$` at the end of each of these?

yshu95 replied:
Thank you for your reply.

As you said, it was possible to just erase the $$ end of the line. 
e.g.   ^\s+${PORT}?\s+\d+?\s+${NAME}?\s+${SN} -> Record

Should I fix it?

FragmentedPacket replied:
I believe so and revert the existing test files and then add your own example to make sure all use cases are covered.

yshu95 commented:
Well, I fixed the original test and added a new example.
But the test does not pass because https://github.com/networktocode/ntc-templates/issues/794.

yshu95 commented:
Well, I fixed the original test and added a new example.

jmcgill298 commented:
Thanks @yshu95
